tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot

```diff
@@ -420,7 +421,7 @@ Container network - space in network name valid
     ${out}=  Run Keyword If  '%{HOST_TYPE}' == 'VC'  Add VC Distributed Portgroup  test-ds  'VM Network With Spaces'
 
     Log To Console  Create a bridge portgroup.
-    ${out}=  Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Run  govc host.portgroup.add -vswitch vSwitchLAN bridge
+    ${out}=  Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Run  govc host.portgroup.add -vswitch -vlan=196 vSwitchLAN bridge
```

Comment:
This should **not** be vlan=196 - that's the "public" port group ID. This is ESX but that is just confusing.